src/assets/scss/_tables.scss

```diff
@@ -86,8 +86,6 @@ table.confluenceTable th.confluenceTh {
 }
 
 /* Background colors */
```

Comment:
table.confluenceTable th.confluenceTh:not([data-highlight-colour]),
table.confluenceTable th.confluenceTh:not([data-highlight-colour]) > p {
  background-color: var(--highlight-grey);
}